### DISCUSSION TITLE
NFC Support

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [ionic-team/capacitor#6848] [Q&A]
I haven't found any documentation or package to read an nfc tag like Mifare Desfire?
Are there any plugins out there?

**Comment by robingenz** [comment]
Yes, you can read nfc tags using the [`@capawesome-team/capacitor-nfc`](https://capawesome.io/plugins/nfc/) package for example.

**Disclaimer**: I am the author.

**Comment by NilsBaumgartner1994** [comment]
Love that your plug-in works. A demo app would be nice to see if and how it works.Beste Grüße,Nils BaumgartnerAm 31.08.2023 um 22:11 schrieb Robin Genz ***@***.***>:﻿
Yes, you can read nfc tags using the @capawesome-team/capacitor-nfc package for example.
Disclaimer: I am the author.

**Reply by robingenz** [reply]
You can find the demo app [here](https://github.com/capawesome-team/capacitor-nfc-demo).

**Comment by NilsBaumgartner1994** [comment]
iOS device :-/Beste Grüße,Nils BaumgartnerAm 31.08.2023 um 22:18 schrieb Robin Genz ***@***.***>:﻿
You can find the demo app here.
